### PR TITLE
update using-javascript-transforms with new layout graphql

### PR DESCRIPTION
**Pull request by jbolda**

### CODE REVIEW

File: examples/using-javascript-transforms/gatsby-node.js [hunk 1]
```diff
@@ -70,6 +71,12 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           reject(result.errors)
         }
 
+        const layoutIndex = path.resolve(`src/layouts/index.js`)
```
Comment (KyleAMathews) [inline examples/using-javascript-transforms/gatsby-node.js]: This isn't necessary. Any component in `src/layouts` automatically becomes a layout. You only use `createLayout` directly if you're creating layouts in other places.

File: examples/using-javascript-transforms/gatsby-node.js [hunk 2]
```diff
@@ -83,6 +90,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
             createPage({
               path: frontmatter.path, // required
               component: markdownTemplate,
+              layout: `index`,
```
Comment (KyleAMathews) [inline examples/using-javascript-transforms/gatsby-node.js]: Not required to specify the index specifically as the index layout is set by default.

**Comment by KyleAMathews** [comment]
This is ready to go?

**Comment by jbolda** [comment]
Ready to go.

File: examples/using-javascript-transforms/gatsby-node.js [hunk 3]
```diff
@@ -23,9 +23,10 @@ exports.onCreateNode = ({ node, boundActionCreators, getNode }) => {
 }
 
 exports.createPages = ({ graphql, boundActionCreators }) => {
-  const { createPage } = boundActionCreators
+  const { createPage, createLayout } = boundActionCreators
```
Comment (KyleAMathews) [inline examples/using-javascript-transforms/gatsby-node.js]: Could you remove the `createLayout` real quick if you're around?

Comment (jbolda) [inline examples/using-javascript-transforms/gatsby-node.js]: done

**Comment by KyleAMathews** [comment]
👍

**Pull request merged**